integration_test/build-adapters.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+
+build_adapter () {
+  adapter_name=${base_name}${1}
+  version=$(cat build.gradle | grep "version = '"| grep -Eo "[0-9]{1,3}\.[0-9]{1,3}")
```

Comment:
Can we move the version from the gradle build file into a standalone file called VERSION? This file can contain the first two version numbers: X.Y
There are gradle commands which can be used to read the file. That way it's easy for someone to update/check the version, rather than having to look into the gradle.build files.

Reply:
>That way it's easy for someone to update/check the version

this is may be will make it easy for someone who is not familiar with gradle but definitely will make it confusing for someone who used to work with gradle, so it's controversial.
>There are gradle commands which can be used to read the file. 

i may suggest to add something like 

`
task printVersion {
  println project.version
}
`
but output won't be clear and we'll need to grep it as well
![image](https://user-images.githubusercontent.com/878990/64247671-8a300880-cf39-11e9-9e60-e18348dc8e8d.png)

Reply:
responded here https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/pull/32#discussion_r321152342

Reply:
done